docs/biblioteca_aserciones.md

```diff
@@ -0,0 +1,44 @@
+# Biblioteca de Aserciones
+
+Para la elección de la biblioteca de aserciones, he buscado y evaluado las distintas alternativas posibles. He buscado herramientas en [Snyk Advisor](https://snyk.io/advisor/) y en foros de Reddit como [Is there life beyond PyUnit/PyTest?](https://www.reddit.com/r/Python/comments/1gbxrho/anyone_working_on_or_with_a_great_but_littleknown/) y [Anyone working on or with a great but little-known test framework? ](https://www.reddit.com/r/Python/comments/1gbxrho/anyone_working_on_or_with_a_great_but_littleknown/). A continuación detallo los criterios específicos que he tenido en cuenta para la elección de la herramienta. 
+
+- Se considerarán las herramientas que tengan una buena puntuación en [Snyk Advisor](https://snyk.io/advisor/).
+- Se considerarán las herramientas que tengan soporte activo. Podemos verificarlo viendo en las páginas de [Snyk Advisor](https://snyk.io/advisor/) y [PyPi](https://pypi.org/) si las últimas versiones disponibles de cada herramienta son recientes.  
+- Se considerará que la herramienta ofrezca una amplia gama de opciones para realizar diferentes tipos de aserciones. Esto evitará deuda técnica innecesaria en el futuro, permitiendo escribir pruebas específicas.  
```

Comment:
Si ya hemos hablado de aserciones tipo BDD y los otros tipos y se puede ver de forma objetiva, ¿por qué no usáis eso?
También deberías hablar de tipos de aserciones específicos, por ejemplo, ¿se pueden testear excepciones?